src/proc/namer.rs

```diff
@@ -102,7 +104,12 @@ impl Namer {
             }
             None => {
                 let mut suffixed = base.to_string();
-                if base.ends_with(char::is_numeric) || self.keywords.contains(base.as_ref()) {
+                if base.ends_with(char::is_numeric)
+                    || self.keywords.contains(base.as_ref())
+                    || self
+                        .keywords_case_insensitive
+                        .contains(&AsciiUniCase(base.to_owned().into_owned()))
```

Comment:
This will allocate unnecessarily, is there anything missing to get the suggestion working?

```suggestion
                        .contains(&AsciiUniCase(base))
```

Reply:
I tried messing around with this (basically doing a `String` -> `&str` type conversion), but as far as I understand of Rust that would require unsafe code (that I'm not even sure would be stable). We'd need to implement `Borrow<T>` somehow which means we'd need to go `AsciiUnicase<String>` -> `&AsciiUnicase<str>`.

Reply:
I think we can instead use a `&'static str` type since all those keywords are 'static (or `Cow<'static, str>` if we need to).
```rust
keywords_case_insensitive: FastHashSet<AsciiUniCase<&'static str>>,
```